km_api/know_me/tests/models/test_km_user_accessor_model.py

```diff
@@ -20,4 +20,5 @@ def test_string_conversion(km_user_accessor_factory):
     """
     accessor = km_user_accessor_factory()
 
-    assert str(accessor) == "Accessor for " + accessor.km_user.name
+    expected = 'Accessor for {user}'.format(user=accessor.km_user.name)
```

Comment:
So in the test you assert for `km_user.name`, but the implementation essentially returns `str(km_user)`. This is fine, and works right now, but what is the intended behavior? Do you want it to always contain the Know Me user's `name` property, or do you want it to contain the string representation of the user?